src/mozilla_sec_eia/models/sec10k/utils/cloud.py

```diff
@@ -162,35 +160,22 @@ class GoogleCloudSettings(BaseSettings):
     MLFLOW_TRACKING_URI: URI of mlflow tracking server.
     """
 
-    model_config = SettingsConfigDict(env_file=".env")
-
-    filings_bucket_name: str = Field(validation_alias="GCS_FILINGS_BUCKET_NAME")
-    labels_bucket_name: str = Field(validation_alias="GCS_LABELS_BUCKET_NAME")
-    metadata_db_instance_connection: str = Field(
-        validation_alias="GCS_METADATA_DB_INSTANCE_CONNECTION"
-    )
-    user: str = Field(validation_alias="GCS_IAM_USER")
-    metadata_db_name: str = Field(validation_alias="GCS_METADATA_DB_NAME")
-    project: str = Field(validation_alias="GCS_PROJECT")
-    tracking_uri: str = Field(validation_alias="MLFLOW_TRACKING_URI")
-
-
-class GCSArchive(BaseModel):
-    """Provides an interface for archived filings on GCS."""
-
-    settings: GoogleCloudSettings = Field(default_factory=lambda: GoogleCloudSettings())
+    filings_bucket_name: str
```

Comment:
hadn't really thought about it but the `ConfigurableResource` is much nicer than the `BaseSettings`, at least in a Dagster context

Reply:
Yeah they're super useful! No more passing around settings classes all over the place